examples/directml/stable_diffusion_xl/stable_diffusion_xl.py

```diff
@@ -267,14 +267,6 @@ def optimize(
         with open(script_dir / f"config_{submodel_name}.json", "r") as fin:
             olive_config = json.load(fin)
 
-        # TODO: Remove this once we figure out which nodes are causing the black screen
-        if is_refiner_model and submodel_name == "vae_encoder":
```

Comment:
Your fix only targets vae_decoder right? So the issue in vae_encoder should still be there. We don't need to address vae_encoder in this PR, but we should keep that part disabled.

Reply:
skipped the nodes in vae_decoder fixed the black screen issue, so vae_encoder should not be an issue anymore.

Reply:
Did you test it with the refiner model, or only SDXL?